src/db/params.rs

```diff
@@ -71,16 +70,16 @@ bso_data! {
     DeleteBso {},
     GetBso {},
     PutBso {
-        sortindex: Option<i64>,
+        sortindex: Option<i32>,
         payload: Option<String>,
-        ttl: Option<i64>,
+        ttl: Option<u32>,
```

Comment:
The API docs define `ttl` as unsigned with up to 9 digits, so I switched this to `u32` everywhere.

Ref: https://mozilla-services.readthedocs.io/en/latest/storage/apis-1.5.html#basic-storage-object